SebastianHerrera/bootcamp-java/src/main/java/hackerrank/day3/list/ExerciseList.java

```diff
@@ -0,0 +1,37 @@
+package hackerrank.day3.list;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * @author Sebastian Herrera (bermeosebas@gmail.com)
+ * @created 31/08/21
+ */
+public class ExerciseList {
+
+    public static final String INSERT = "Insert";
+    public static final String DELETE = "Delete";
+
+    public static void getListInfo() {
+        Scanner scanner = new Scanner(System.in);
+        int numElementsList = scanner.nextInt();
+        List<Integer> dataList = new ArrayList<>();
+        for (int index = 0; index < numElementsList; index++) {
+            dataList.add(scanner.nextInt());
+        }
+        int numberQueries = scanner.nextInt();
+        for (int index = 0; index < numberQueries; index++) {
```

Comment:
I think somebody is peeking at the code of their partners.....

Reply:
Nooo,  I think great minds think alike. :woozy_face: